app/models/game.rb

```diff
@@ -2,6 +2,8 @@ class Game < ActiveRecord::Base
   has_many :pieces
   delegate :pawns, :knights, :bishops, :rooks, :queens, :kings, to: :pieces
 
+  attr_accessor :winner_id
```

Comment:
you probably want the winner id to actually be a field in the database not just an attr_accessor